File: fizzbuzz.py

x = 1

while x <= 100:
	flag = 0
	
	three = 0
	if (x % 3) == 0: three = 1
	
	five = 0
	if (x % 5) == 0: five = 1
	
	flag += three * 1
	flag += five * 2
	
	if flag == 0:
		print(x)
	elif flag == 1:
		print('Fizz')
	elif flag == 2:
		print('Buzz')
	elif flag == 3:
		print('FizzBuzz')
	x += 1

File: fizzbuzz.lua

-- fizzbuzz 

for x = 1, 100 do
  
  flag = 0
  three = 0
  five = 0
  if (x % 3) == 0 then three = 1 end
  if (x % 5) == 0 then five = 1 end
  
  flag = flag + three * 1
  flag = flag + five * 2
  
  if flag == 0 then print(x) 
  elseif flag == 1 then print('fizz')
  elseif flag == 2 then print('buzz')
  elseif flag == 3 then print('fizzbuzz')
  end
  
end
	
	
